src/backend/cdb/cdbquerycontextdispatching.c

```diff
@@ -102,7 +102,7 @@ int QueryContextDispatchingSizeMemoryLimit = 100 * 1024; /* KB */
 
 enum QueryContextDispatchingItemType
 {
-    MasterXid, TablespaceLocation, TupleType, EmptyTable, FileSystemCredential
+    MasterXid, TablespaceLocation, TupleType, EmptyTable, FileSystemCredential, Namespace
```

Comment:
can you please explain this part? does it mean the namespace becomes part of the dispatched data? if so, could we potentially get it without changing the external table URL?